processes/github_access.md

```diff
@@ -20,8 +20,13 @@ All projects at Defra must be created under one of the organisations within the
 - [Defra](https://github.com/DEFRA) for software
 - [Defra Data Science Centre of Excellence](https://github.com/-Defra-Data-Science-Centre-of-Excellence) for data science
 - [Defra design team](https://github.com/defra-design) for prototype designs
+- [aphascience](https://github.com/aphascience) for the APHA Scientific Computing Environment
 
-The organisation owners can be contacted to add you to their organisation.
+The organisation owners can be contacted to add you to their organisation:
+
+- for software development or design, use the #github-support channel on the Defra Digital Slack workspace
```

Comment:
A bit like the comment above, regarding "software"... if we start directing our people here for all GitHub usage standards they might get confused that they should be contacting the wrong support.

Reply:
Amended to say digital service development as hopefully that clarifies it a bit